.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 # v3.5.0
         with:
-          go-version: "1.19.5"
+          go-version: "1.20.14"
```

Comment:
Is the dot release actually needed here?

Reply:
Not sure, to be honest. It was a quick fix so I just copied the same pattern. Let me check.

Reply:
It seems it's not needed: https://github.com/marketplace/actions/setup-go-environment#supported-version-syntax Let's see in practice

Reply:
It works :+1: